compute_tools/src/compute_prewarm.rs

```diff
@@ -28,7 +20,7 @@ struct EndpointStoragePair {
 
 const KEY: &str = "lfc_state";
 impl EndpointStoragePair {
-    /// endpoint_id is set to None while prewarming from other endpoint, see replica promotion
+    /// endpoint_id is set to None while prewarming from other endpoint, see compute_promote.rs
```

Comment:
Here we could have probably referred directly to a module instead using rustdoc, couldn't we?